dataactvalidator/config/sqlrules/c5_award_financial_2.sql

```diff
@@ -17,6 +17,12 @@ SELECT
     uri AS "uniqueid_URI"
 FROM award_financial
 WHERE submission_id = {0}
+    AND gross_outlay_amount_by_awa_cpe IS NOT NULL
```

Comment:
This was discussed with @schichester .  The wording of this may need updated in a future release but `GrossOutlayAmountByAward_CPE` *can* be zero and still fine. Sending the discussion over slack.